cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/TheirDetailsMapper.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+public class TheirDetailsMapper {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty class, write its code or make it an "interface". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS2094)